applications/openshift/etcd/etcd_backup/rule.yml

```diff
@@ -0,0 +1,29 @@
+documentation_complete: true
+
+title: Configure Recurring Backups For etcd
+
+description: |-
+    <p>
+    Back up your clusters etcd data regularly and store in a secure location ideally outside the OpenShift Container Platform environment. Do not take an etcd backup before the first certificate rotation completes, which occurs 24 hours after installation, otherwise the backup will contain expired certificates. It is also recommended to take etcd backups during non-peak usage hours because the etcd snapshot has a high I/O cost.
+    </p>
+    <p>
+    For more information, follow
+    {{{ weblink(link="https://docs.openshift.com/container-platform/latest/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html#backing-up-etcd-data_backup-etcd",
+                text="the relevant documentation") }}}.
+    </p>
+
+rationale: |-
+    While etcd automatically recovers from temporary failures, issues may arise if an etcd cluster loses more than (N-1)/2 or when an update goes wrong.
+    Recurring backups of etcd enable you to recover from a disastrous fail.
+
+identifiers: {}
```

Comment:
Please, ensure that the new rules included in the BSI profile have an identifier.

Run the following:
```sh
. .pyenv.sh
./utils/rule_dir_json.py # this will create a json file that used by the next python script
python utils/fix_rules.py --product products/ocp4/product.yml add-cce --cce-pool redhat etcd_backup general_backup_solution_installed # this grabs a CCE from Red Hat's pool and assigns it to the rule
# Edit the files to remove the empty set and fix indentation
git add -u
git add shared/references/cce-redhat-avail.txt
git commit
```
At the end you should have something like: https://github.com/ComplianceAsCode/content/blob/0a1ba9385c4daf6816a74e730a11597c0bb8e41d/applications/openshift/etcd/etcd_key_file/rule.yml#L30-L31